homeassistant/components/vicare/strings.json

```
@@ -242,6 +242,44 @@
       "volumetric_flow": {
         "name": "Volumetric flow"
       },
+      "ess_state_of_charge": {
+        "name": "Battery"
+      },
+      "ess_power_current": {
+        "name": "Battery power"
+      },
+      "ess_state": {
+        "name": "Battery state",
+        "state": {
+          "charge": "Charge",
+          "discharge": "Discharge",
+          "standby": "Standby"
+        }
+      },
+      "pcc_current_power_exchange": {
+        "name": "PCC power exchange"
+      },
+      "pcc_energy_consumption": {
+        "name": "PCC energy consumption"
+      },
+      "pcc_energy_feed_in": {
+        "name": "PCC energy feed-in"
+      },
```

What's a PCC?

---

*Point of common coupling*

---

Is this a common ViCare term? Would using Grid maybe make more sense in the naming?

---

No, had to look it up, but seems to be common in electrical area.

> As per IEEE, the PCC can be defined as the point in the power system at which the electric utility and the customer interface occurs. Typically this point is the customer side of the utility revenue meter.

https://www.sciencedirect.com/topics/engineering/point-of-common-coupling#:~:text=3.1%20Point%20of%20common%20coupling,of%20the%20utility%20revenue%20meter.

What about this?
```suggestion
      "pcc_current_power_exchange": {
        "name": "PCC power exchange"
      },
      "pcc_energy_consumption": {
        "name": "Energy import from grid"
      },
      "pcc_energy_feed_in": {
        "name": "Energy export to grid"
      },
```

---

Sounds good, what about the power exchange?